Makefile

```diff
@@ -45,3 +46,16 @@ docker-build: fmt vet
 docker-push:
 	docker push ${IMG}
 
+build-svc:
+	go build -v -o bin/app ./services/${SVC}/ && \
```

Comment:
I don't think we should build the binary outside of the container runtime. It feels like an opening for something to go wrong. 

Off the top of my head my issues are:

1. If the user has a docker remote host that differs in GOARCH and GOOS from their workstation (as I do) this can cause issues
2. There might be a version difference from workstation golang to container golang

Reply:
( to expand, I'm running darwin/arm but my docker host is linux/amd64 )

Reply:
Change it to use the Dockerfile of the services and therefore building inside the Container Runtime